Marlin/src/feature/bedlevel/ubl/ubl.cpp

```diff
@@ -150,7 +150,7 @@
     SERIAL_ECHO_SP(7);
     LOOP_L_N(i, GRID_MAX_POINTS_X) {
       if (i < 10) SERIAL_CHAR(' ');
-      SERIAL_ECHO((int)i); //PLEASE LEAVE THIS COMMENT, THIS CODE IS HARD TO FIND, I'LL REVERT IT LATER ON
+      SERIAL_ECHO((int)i);
```

Comment:
Yes, it is for now. I'm sorry, but those are hard to find in the code. The next step is to revert the print(uint8) to actually print the number (not the ASCII value). 

When this is done, I'll remove the cast as it'll not be required anymore, so I need some easy to find message to find them (I don't want to search for all `(int)` cast in the code.

For this to work, all the other place where SERIAL_ECHO(uint8) is used instead of SERIAL_CHAR will have to be converted to the later.
When this is done, I'll have a clearer usage of the right serial macro.

Reply:
I've spread the exact same message in multiple places, so if you revert one, you'll have to revert all. I'd prefer you don't revert any.